x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/ColumnInfoImpl.java

```diff
@@ -132,4 +132,8 @@ public DataType type() {
     public List<String> originalTypes() {
         return originalTypes;
     }
+
+    public String toString() {
```

Comment:
Not required for the fix strictly speaking, but it improves observability of columns and makes it much easier to inspect them when debugging.